### PR TITLE
Account signup and delete test

### PR DESCRIPTION
**Pull request by n1313**
This PR adds a test for account signup and deletion. Note: this won't work on candy/prod without a way to circumvent captcha.

### CODE REVIEW

**Comment by n1313** [comment]
(assumes https://github.com/FreeFeed/freefeed-react-client/issues/1281 to be fixed)

File: cypress.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "https://candy.freefeed.net",
+  "baseUrl": "http://localhost:3333/",
```
Comment (Spaceinvaderz) [inline cypress.json:2]: small concern here. probably case with running against local setup of FrF should be described in readme in this repo (or added some links at least).

**Review by Spaceinvaderz — APPROVED** [review]

File: cypress.json (hunk 1 above)
Comment (n1313) [reply cypress.json:2]: will make readme clearer on this in a separate PR

Comment (n1313) [reply cypress.json:2]: Done https://github.com/FreeFeed/acceptance-tests/commit/a504f80103dd2c795030c191f64865c6bcec2afb

**Pull request merged**